public_html/includes/classes/forms.php

```diff
@@ -1137,7 +1137,7 @@ public static function buildEditTable($formID) {
 						);
 
 				foreach ($form['fields'] as $field) {
-					$temp[] = sprintf('<input type="%s" style="%s" name="%s_%s" value="%s" readonly />',
+					$temp[] = sprintf('<textarea type="%s" style="%s" name="%s_%s" readonly>%s</textarea>',
```

Comment:
From my understanding it was tested and the only part of the MFCS that changes is the meta-data forms.  If any other parts change that we are un-aware of with this change please inform by naming specific items that are broken.  Speculation does no good with out constructive comments.  

Looking at the code it does need modified to have an if condition on the type.  
If the type equals text then it could be changed to a textarea, otherwise let it remain as a readonly input for items such as checkboxes, dates, etc.  

`$temp[] = sprintf('<input type="%s" style="%s" name="%s_%s" value="%s" readonly />',`
Readonly being that the code will never be submitted back so if read only how will it break the form.  I would also like to know why it is a form that is readonly with nothing submitted back.  At one point we did try to do ajax fields for items that have changed, but the default functionality for that didn't work so the edit button moves them to another meta-data edit form.